chef-utils/spec/spec_helper.rb

```diff
@@ -21,6 +22,7 @@
 OS_HELPERS = (ChefUtils::DSL::OS.methods - Module.methods).freeze
 PLATFORM_FAMILY_HELPERS = (ChefUtils::DSL::PlatformFamily.methods - Module.methods).freeze
 PLATFORM_HELPERS = (ChefUtils::DSL::Platform.methods - Module.methods).freeze
+RENDER_HELPERS = (ChefUtils::DSL::RenderHelpers.methods - Module.methods).freeze
```

Comment:
I know this is copying an existing pattern, but I feel like it's kind of overkill to define a constant in spec_helper that is used in one test file.

Reply:
yeah i did it centralized though because otherwise people will probably copypasta around the constants like I did initially and defining them on the top level will get constant redefinition warnings and will break stuff.

Reply:
(it ain't good, but the alternative i had first was worse and this at least makes it obvious you need unique names, but that's about all)